apps/emqx_message_transformation/src/emqx_message_transformation.app.src

```diff
@@ -0,0 +1,14 @@
+{application, emqx_message_transformation, [
+    {description, "EMQX Message Transformation"},
+    {vsn, "0.1.0"},
+    {registered, [emqx_message_transformation_sup, emqx_message_transformation_registry]},
+    {mod, {emqx_message_transformation_app, []}},
+    {applications, [
+        kernel,
+        stdlib
```

Comment:
Need to depend on `emqx` to use hooks?

Reply:
It's a dependency in `rebar.config`, but the app should be started, indeed.

Reply:
:heavy_check_mark: